data/sql_updates/sched_a_by_state_recipient_totals.sql

```diff
@@ -0,0 +1,86 @@
+-- Schedule A summary of individual contributions by state raised for each committee type per cycle.
+drop materialized view if exists ofec_sched_a_aggregate_state_recipient_totals_mv_tmp;
+
+create materialized view ofec_sched_a_aggregate_state_recipient_totals_mv_tmp as
+with grouped_totals as (
+    select
+        sum(agg_st.total) as total,
+        count(agg_st.total) as count,
+        agg_st.cycle,
+        agg_st.state,
+        agg_st.state_full,
+        cd.committee_type,
+        cd.committee_type_full
+    from
+        ofec_sched_a_aggregate_state as agg_st
+    join
+        ofec_committee_detail_mv_tmp as cd
+    on
+        agg_st.cmte_id = cd.committee_id
+    where
+        agg_st.state in (
+            select
+                "Official USPS Code"
+            from
+                ofec_fips_states
+
+            -- NOTE:  If we ever need to account for the FIPS numeric codes,
+            --        we can do so with this subquery instead.
+
+            --select
+            --    "FIPS State Numeric Code"::text as state_codes
+            --from
+            --    ofec_fips_states
+            --union all
+            --select
+            --    "Official USPS Code"::text as state_codes
+            --from
+            --    ofec_fips_states
+        )
+    group by
+        agg_st.cycle,
+        agg_st.state,
+        agg_st.state_full,
+        cd.committee_type,
+        cd.committee_type_full
+),
+overall_total as (
```

Comment:
D'oh!  Thanks for the catch, adding two more groupings to account for these totals.